services/ops/TopicWatcher/README.md

```diff
@@ -0,0 +1,52 @@
+## Topic Watcher Agent
+
+The Topic Watcher Agent listens to a set of configured topics and publishes an alert if they are not published within 
+some time limit.  In addition to “standard” topics the Topic Watcher Agent supports inspecting device "all" topics. This
+can be useful when a device contains volatile points that may not be published.
+
+
+### Requirements
+
+The Topic Watcher agent requires the Sqlite 3 package. This package can be installed in an activated environment with:
```

Comment:
It is also already included with various bootstrap flags. Is there one or more we should reference that should be used when this agent is anticipated?